README.md

```diff
@@ -1,4 +1,87 @@
-# openmrs-esm-patient-registration
 [![Build Status](https://travis-ci.org/openmrs/openmrs-esm-patient-registration.svg?branch=master)](https://travis-ci.org/openmrs/openmrs-esm-patient-registration)
 
-[Wiki documentation](https://wiki.openmrs.org/display/projects/openmrs-esm-patient-registration)
+# Patient Registration ESM
```

Comment:
I edited the README file directly on GitHub on the master branch. So that would probably be the reason why. The master has the most up to date version of the README

Reply:
Hm no I think you didn't start from `master`. The changeset indicates that the two branches are diverged - which, since we do squash merges, indicates that the current branch has not been started from the former `master`.